x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/columns.tsx

```diff
@@ -64,6 +64,14 @@ export const getTaskStateBadge = (
   );
 };
 
+export const getJobTypeBadge = (jobType: string) => {
```

Comment:
Nit: this could just be an implicit return since no other logic is happening in this function.

Reply:
Good point! Changed this to implicit return in  https://github.com/elastic/kibana/pull/65770/commits/6c4abae25cd87c3e2b455a641451f88e826745b3.